codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/customizations/SmithyTypesPubUseExtra.kt

```diff
@@ -26,6 +26,14 @@ private fun hasStreamingOperations(model: Model): Boolean {
     }
 }
 
+/** Returns true if the model has event streaming operations */
+private fun hasEventStreamOperations(model: Model): Boolean =
```

Comment:
Can you use https://github.com/awslabs/smithy-rs/blob/847ed0b66afe6eca3c64a764a4e7c7c16b0851df/codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/util/Smithy.kt#L93? The codegen context has the service shape in it.

Reply:
Addressed in 91163de.